docs/src/tutorials/basics.md

```diff
@@ -192,7 +192,7 @@ get_forecast_initial_times(sys)
 We can find the names of all time series attached to a component:
 
 ```@repl basics
-ts_names = get_time_series_names(Deterministic, loads[1])
+show_time_series(loads[1])
```

Comment:
I want to emphasize the fact that I'm proposing this to be the way users view time series for a component. I have removed time series display from `Base.show(component)`. I can make a good case for this, but you may disagree and we can talk about it.